27/mars_documentation.md

```diff
@@ -10,19 +10,41 @@ MARS is comprised of distinct components, each representing key roles in the dat
 Each role performs specific functions and carries distinct responsibilities essential for the smooth operation of the system.
 
 ### ISA-JSON producers
-This role includes research infrastructure or service facilities that operate platforms such as ARC, FAIRDOM-SEEK, ISA Creators or any other tool that generates ISA-JSON files.
-* Function: Prepare research data in ISA-JSON format, generate and export ISA-JSON files.
+This role includes any source of ISA-JSON files containing the metadata of multi-omics research. For example, research infrastructure or service facilities that operate platforms such as ARC, FAIRDOM-SEEK, ISA Creators. 
+
+* Main function: Prepare research metadata in ISA-JSON format (i.e. generate and export ISA-JSON files)
 * Responsibilities:
-  * Ensure compatibility of the exported ISA-JSON files with the format expected by MARS.
-  * Support and validate for the use of repositories’ checklists, controlled vocabulary, ontology terms, and other required metadata.
+  * Avoid data duplication. For example, not re-submitting already existing data in the target repositories through MARS.
+  * Ensure compliance of the submitted ISA-JSON with the ISA model expected by MARS.
+  * Conform submitted ISA-JSON to target repositories' additional validation rules validated by MARS. This includes repositories’ checklists, controlled vocabulary lists, ontology terms, etcetera.
```

Comment:
Is MARS going to do the validation of checklists etc? I thought that would be done by target archives during submission?